Basic-Car-Maintenance/Shared/Odometer/Views/OdometerView.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  OdometerView.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Mikaela Caron on 10/11/23.
+//
+
+import SwiftUI
+
+struct OdometerView: View {
+    @Environment(ActionService.self) var actionService
+    @Environment(\.scenePhase) var scenePhase
+    @Bindable private var viewModel: OdometerViewModel
```

Comment:
I think the viewModel should be `State` not `Bindable` because it's being created here

Reply:
@mikaelacaron TBH I don't have a strong understanding of the difference. When I change @Bindable to @State, I have to rewrite the logic in the init function for the ViewModel, but I haven't quite figured it out.

Reply:
Looks like it wasn't an issue after all? Fixed in https://github.com/mikaelacaron/Basic-Car-Maintenance/pull/175/commits/d3b5d69d7f3e5150cb3b66fa2ebac4bbec96680c